tests/python/gaia-ui-tests/gaiatest/apps/contacts/regions/contact_form.py

```diff
@@ -114,8 +117,13 @@ def type_comment(self, value):
         element.send_keys(value)
 
     def tap_comment(self):
-        self.marionette.find_element(*self._add_new_note_locator).tap()
+        element = self.marionette.find_element(*self._add_new_note_locator)
+        element.tap()
+        self.marionette.execute_script(
+            'arguments[0].scrollIntoView(true);', [element])
```

Comment:
Afaik, there isn't http://mxr.mozilla.org/gaia/search?string=scrollintoview&find=.py
But I'll file a bug about this.